test/services/XmlValidationServiceSpec.scala

```diff
@@ -0,0 +1,132 @@
+/*
+ * Copyright 2020 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package services
+
+import base.SpecBase
+
+class XmlValidationServiceSpec extends SpecBase {
```

Comment:
This service is testing against a specific XSD, which means that anyone looking/maintaining at this test will need to look at that XSD, understand it and construct valid XML sample for that XSD. Instead, should we have a simple XSD that we can easily construct XML data for and which is purposely designed to be simple?